lib/cli.js

```diff
@@ -100,6 +103,11 @@ exports.run = function () {
 		throw new Error(colors.error(figures.cross) + ' The --require and -r flags are deprecated. Requirements should be configured in package.json - see documentation.');
 	}
 
+	if (cli.flags.extension) {
+		cli.flags.extension = arrify(cli.flags.extension);
+		cli.flags.extension.push('js');
```

Comment:
As we discussed in https://github.com/avajs/ava-files/pull/9#issuecomment-261962080, the option should override `js`.